src/Components/Artwork/Details/Details.tsx

```diff
@@ -207,7 +207,8 @@ const CollectorSignalLine: React.FC<DetailsProps> = ({
 
   return (
     <Text
-      variant="xs"
+      variant="sm-display"
+      lineHeight="22px"
```

Comment:
I know its mentioned that things might revert up above, but: when we start messing with typography like this (especially on large surfaces like the artwork brick) we break the lego-block uniformity of the design system, which has its own baked in line-heights right inside each typography variant. Adjustments like this should only be a last resort.